BookingsApi/BookingsApi/Controllers/HearingsController.cs

```diff
@@ -509,8 +509,13 @@ private async Task UpdateHearingStatusAsync(Guid hearingId, BookingStatus bookin
         [ProducesResponseType(typeof(BookingsResponse), (int)HttpStatusCode.OK)]
         [ProducesResponseType((int)HttpStatusCode.BadRequest)]
         [ProducesResponseType((int)HttpStatusCode.NotFound)]
-        public async Task<ActionResult<BookingsResponse>> GetHearingsByTypes([FromQuery(Name = "types")]List<int> types, [FromQuery]string cursor = DefaultCursor, [FromQuery]int limit = DefaultLimit)
+        public async Task<ActionResult<BookingsResponse>> GetHearingsByTypes([FromQuery(Name = "types")]List<int> types, [FromQuery]string cursor = DefaultCursor, [FromQuery]int limit = DefaultLimit, [FromQuery] DateTime? fromDate = null)
         {
+            if (!fromDate.HasValue) {
+                var utcNow = DateTime.UtcNow;
+                fromDate = new DateTime(utcNow.Year, utcNow.Month, utcNow.Day);
```

Comment:
Were there any issues when you tried to initialize fromDate with DateTime.UtcNow? By default it should just give 00:00:00 for the time